f2/src/forms/MoneyLostInfoForm.js

```diff
@@ -69,6 +69,12 @@ export const MoneyLostInfoForm = (props) => {
           <Trans id="transactionDate.error.hasNoMonth" />
           <Trans id="transactionDate.error.hasNoDay" />
           <Trans id="moneyLostPage.financialTransactions" />
+          <Trans id="previousPageofMoneyLost.nextPage" />
```

Comment:
We should avoid doing this whenever possible.

Reply:
Ya, agree.  I tried to put it where it defined, but it didn't work. Do you know a better way to do it? Thanks!

Reply:
As far as I know this is a limitation of Lingui. We are declaring what message property to use outside of React. To avoid this we would have to set the label with React in scope. Since we have this type of thing everywhere this isn't a show stopper, just thought I would point it out. This type of thing would probably be better resolved in a separate issue.